src/main/kotlin/app/revanced/cli/Main.kt

```diff
@@ -129,6 +134,12 @@ class Main {
                 shortName = "o",
                 description = "Output directory"
             ).required()
+            val downloadSignatures by parser.option(
+                ArgType.Boolean,
+                fullName = "downloadSignatures",
```

Comment:
Should be kebab-case.
```suggestion
                fullName = "download-signatures",
```